### PR TITLE
Fix showing JSON columns in Svelte Table

### PR DESCRIPTION
**Pull request by andresgutgon**
## Before
image

## After
image

### CODE REVIEW

File: packages/query_result/src/index.ts [hunk 1]
```diff
@@ -83,7 +83,8 @@ export default class QueryResult {
   toArray() {
     return this.rows.map((row: unknown[]) =>
       row.reduce((acc: Record<string, unknown>, value, i) => {
-        acc[this.fields[i]!.name] = value
+        acc[this.fields[i]!.name] =
+          typeof value === 'object' ? JSON.stringify(value) : value
```
Comment (andresgutgon) [inline packages/query_result/src/index.ts:87]: Can this brake any use of `toArray`?

**Review by csansoon — DISMISSED** [review]
👌

File: packages/client/svelte/src/lib/internal/ui/Table/Table.svelte [hunk 2]
```diff
@@ -26,11 +29,27 @@
...
+    )
+  }
+
+  $: table = createTable(readable(toArray(data.fields, data)))
   $: columns = table.createColumns(
     data.fields.map((field) =>
-      table.column({ accessor: field.name, header: field.name })
-    )
+      table.column({ accessor: field.name, header: field.name }),
+    ),
```
Comment (csansoon) [inline packages/client/svelte/src/lib/internal/ui/Table/Table.svelte:52]: cuidao el prettier

File: packages/client/svelte/src/lib/internal/ui/Table/Table.svelte [hunk 3]
```diff
@@ -26,11 +29,27 @@
   let className: $$Props['class'] = undefined
   export { className as class }
 
-  $: table = createTable(readable(data.toArray()))
+  /**
+   * This is very similar to QueryResult.toArray
+   * but please resists your sharp dry instintcs and
+   * leave this method here.
+   * We only want to stringify JSON columns in this case
+   */
```
Comment (csansoon) [inline packages/client/svelte/src/lib/internal/ui/Table/Table.svelte:37]: 🙈

File: packages/client/svelte/src/lib/internal/ui/Table/Table.svelte (hunk 2 above)
Comment (andresgutgon) [reply packages/client/svelte/src/lib/internal/ui/Table/Table.svelte:52]: Is all green https://www.instagram.com/p/C8WsbScJYse/?hl=en

**Pull request merged**